src/backend/distributed/deparser/deparse_foreign_server_stmts.c

```diff
@@ -0,0 +1,304 @@
+/*-------------------------------------------------------------------------
+ *
+ * deparse_foreign_server_stmts.c
+ *	  All routines to deparse foreign server statements.
+ *
+ * Copyright (c) Citus Data, Inc.
+ *
+ *-------------------------------------------------------------------------
+ */
+#include "postgres.h"
+
+#include "commands/defrem.h"
+#include "distributed/citus_ruleutils.h"
+#include "distributed/deparser.h"
+#include "distributed/listutils.h"
+#include "distributed/relay_utility.h"
+#include "lib/stringinfo.h"
+#include "nodes/nodes.h"
+#include "utils/builtins.h"
+
+static void AppendCreateForeignServerStmt(StringInfo buf, CreateForeignServerStmt *stmt);
+static void AppendCreateForeignServerOptions(StringInfo buf,
+											 CreateForeignServerStmt *stmt);
+static void AppendAlterForeignServerStmt(StringInfo buf, AlterForeignServerStmt *stmt);
+static void AppendAlterForeignServerOptions(StringInfo buf, AlterForeignServerStmt *stmt);
+static void AppendAlterForeignServerRenameStmt(StringInfo buf, RenameStmt *stmt);
+static void AppendAlterForeignServerOwnerStmt(StringInfo buf, AlterOwnerStmt *stmt);
+static void AppendDropForeignServerStmt(StringInfo buf, DropStmt *stmt);
+static void AppendServerNames(StringInfo buf, DropStmt *stmt);
+static void AppendBehavior(StringInfo buf, DropStmt *stmt);
+static char * GetDefElemActionString(DefElemAction action);
+
+char *
+DeparseCreateForeignServerStmt(Node *node)
```

Comment:
minor: needs comment

Reply:
Well, it seems we don't have much comments on these deparse functions, so I'm fine with not having here as well